srv/salt/_modules/iscsi.py

```diff
@@ -120,9 +120,9 @@ class CephIscsiConfig(object):
         "emulate_dpo": 1,
         "emulate_fua_read": 1,
         "emulate_fua_write": 1,
-        "emulate_model_alias": 1,
+        "emulate_model_alias": 0,
         "emulate_pr": 1,
-        "emulate_rest_reord": 0,
+        "emulate_rest_reord": 1,
```

Comment:
Where is this emulate_rest_reord value coming from? It should be left zero, as "1" isn't supported by the kernel.

Reply:
Reverted to zero. Thanks.